### PR TITLE
docs: update trusted clusters page

### PR DESCRIPTION
**Pull request by zmb3**

### CODE REVIEW

**Review by stevenGravy — APPROVED** [review]

File: docs/pages/management/admin/trustedclusters.mdx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+is a group of resources connected to a Teleport Auth Server. Each cluster
 manages a separate certificate authority (CA) for all users and resources.
 
 Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly SSH into the Nodes of another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-be running behind a firewall without any ingress ports open.
+seamlessly connect to resources in another cluster, the **leaf cluster**, while
+remaining authenticated with only a single Auth Service. The leaf cluster can be
+running behind a firewall without any ingress ports open.
```
Comment (alexfornuto) [inline docs/pages/management/admin/trustedclusters.mdx:14]: Removing 'be' phrasing. Ref: http://writing.rocks/to-be-or-not-to-be/

```suggestion
remaining authenticated with only a single Auth Service. The leaf cluster can run
behind a firewall without any ingress ports open.
```

File: docs/pages/management/admin/trustedclusters.mdx [hunk 2]
```diff
@@ -449,7 +449,8 @@ not, change `access` to one of your user's roles.
 
 ### Wildcard characters
 
-In role mappings, wildcard characters match any characters in a string.
+In role mappings, the wildcard character `*` can be used to match any number of
```
Comment (alexfornuto) [inline docs/pages/management/admin/trustedclusters.mdx:452]:
```suggestion
In role mappings, the wildcard character `*` is used to match any number of
```

**Review by alexfornuto — APPROVED** [review]
👍 with a couple of minor phrasing suggestions.

File: docs/pages/management/admin/trustedclusters.mdx (hunk 1 above)
Comment (zmb3) [reply docs/pages/management/admin/trustedclusters.mdx]: Great tip, thanks for sharing!

**Pull request merged**